frontend/components/receipts/transfer/enriched-transfer-view.tsx

```diff
@@ -0,0 +1,34 @@
+import { formatNumber, toBigNum } from '@vegaprotocol/utils'
+import { PriceWithSymbol } from '../utils/string-amounts/price-with-symbol.tsx'
+import { VegaKey } from '../../keys/vega-key'
+import { useWalletStore } from '../../../stores/wallets.ts'
+import { ReceiptComponentProps } from '../receipts'
+import { VegaAsset } from '../../../types/rest-api.ts'
+
+export const locators = {
+  enrichedSection: 'enriched-section'
+}
+
+interface EnrichedTransferViewProps extends ReceiptComponentProps {
+  assetInfo: VegaAsset | undefined
+}
+
+export const EnrichedTransferView = ({ transaction, assetInfo }: EnrichedTransferViewProps) => {
+  const { getKeyInfo } = useWalletStore((state) => ({ getKeyInfo: state.getKeyInfo }))
+  const { amount } = transaction.transfer
+  const decimals = Number(assetInfo?.details?.decimals)
+  const price = amount && decimals ? formatNumber(toBigNum(amount, decimals), decimals) : undefined
+  const symbol = assetInfo?.details?.symbol
+  const keyInfo = getKeyInfo(transaction.transfer.to)
+  const isOwnKey = keyInfo?.publicKey === transaction.transfer.to
```

Comment:
nitpick (non-blocking): just simplier, if it exists it is ours so can just do

```suggestion
  const isOwnKey = !!keyInfo
```